src/components/ratings_reviews/Reviews.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReviewListEntry from './ReviewListEntry';
+
+function Reviews() {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        marginLeft: '0.2em',
+        border: 'solid 0px white',
+        padding: '0.2em 0em 0em 0em',
+        'border-radius': '5px',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <em>Reviews</em>
+      <select
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          float: 'right',
+        }}
+      >
+        <option>Sort by...</option>
+        <option>Helpfulness</option>
+        <option>Newest</option>
+        <option>Relevance</option>
+      </select>
+      <br />
```

Comment:
Instead of using the `<br />` at the end, you should be better off adding an additional line in the style object to space out the bottom (I.e. `marginBottom: '1em',`).